tests/web/rpc/v1/test_endpoint_trace_item_table.py

```diff
@@ -0,0 +1,425 @@
+import random
```

Comment:
This is basically a copy over from the test_span_samples tests. all the same cases are covered